src/screens/FeedScreen/index.js

```diff
@@ -257,7 +249,10 @@ const FeedScreen = (props) => {
         refreshing={loading}
         renderItem={renderItem}
         onEndReachedThreshold={0.9}
-        onMomentumScrollEnd={onWillSendViewPostTime}
+        onMomentumScrollEnd={(momentumEvent) => {
+          onWillSendViewPostTime(momentumEvent, feeds);
+          fetchNextFeeds(momentumEvent);
+        }}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onMomentumScrollEnd` prop has been updated to call both `onWillSendViewPostTime` and `fetchNextFeeds`. This is a good change as it allows for both tracking the view time of posts and fetching more feeds when the end of the list is reached. However, make sure that these two functions do not interfere with each other and that they can handle being called simultaneously.


<!-- This is an auto-generated comment by OSS CodeRabbit -->